public/.well-known/assetlinks-testing.json

```diff
@@ -12,8 +12,8 @@
       "delegate_permission/common.handle_all_urls"
     ],
     "target": {
-      "namespace": "PassCulture T",
-      "package_name": "com.passculture",
+      "namespace": "android_app",
```

Comment:
Qu'est-ce que ça fait de changer le namespace ?

Reply:
Android 12 fait un check dessus.

Le `namespace` n'a pas d'autre valeur. https://stackoverflow.com/questions/36897575/where-do-i-find-the-namespace-of-an-app-android.

Ce fichier la n'est pas lu par iOS, on a une autre configuration pour iOS: https://github.com/pass-culture/pass-culture-app-native/blob/master/public/.well-known/apple-app-site-association